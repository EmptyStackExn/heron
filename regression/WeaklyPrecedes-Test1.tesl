h1 weakly precedes h2
@minstep 2
@run
@select 0x36150
@output tikz select h1 h2
