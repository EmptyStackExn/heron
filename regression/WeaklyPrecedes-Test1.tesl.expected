  \node       at (1+0.1, -1+0.1)  {\Cross} ;
  \node       at (1+0.1, 0+0.1)  {\Cross} ;
  \node[tick] at (2, -1)  {} ;
  \node[tick] at (2, 0)  {} ;
